client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

```diff
@@ -2598,6 +2601,35 @@ public void testXPackPutWatch() throws Exception {
         request.getEntity().writeTo(bos);
         assertThat(bos.toString("UTF-8"), is(body));
     }
+    
+    public void testXPackGraphExplore() throws Exception {
```

Comment:
I think maybe this should be `testGraphExplore` now that we're removing the `xpack` in lots of places.